src/rpc/mempool.cpp

```diff
@@ -862,7 +862,7 @@ static RPCHelpMan submitpackage()
         },
         [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
         {
-            if (!Params().IsMockableChain()) {
+            if (Params().GetChainType() != ChainType::REGTEST) {
```

Comment:
This change makes sense independently; `submitpackage` has no relationship to mocktime.